staging/src/k8s.io/kubectl/pkg/cmd/config/delete_user_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+type deleteUserTest struct {
```

Comment:
I see you're following the pattern that was set in this package, but I'd prefer we follow the standards that we have across entire code base. Have a look at this test https://github.com/kubernetes/kubernetes/blob/007f7ae7dc217d27dad994dce003cfe1c5bc6706/staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go#L383-L452 specifically. How it uses anonymous struct to construct the input data and then uses `t.Run` to run actual tests. That's the preferred and canonical way of building tests.